app/config/config.yml

```diff
@@ -32,7 +32,7 @@ framework:
 # Twig Configuration
 twig:
     debug:            "%kernel.debug%"
-    exception_controller: 'FOS\RestBundle\Controller\ExceptionController::showAction'
+    exception_controller: "FOS\\RestBundle\\Controller\\ExceptionController::showAction"
```

Comment:
Not sure, but in Yaml all string should be on double quotes. It fails for sure when string is in double quotes and there is not double backslash.